swiftlang/src/main/java/com/backelite/sonarqube/swift/surefire/SwiftTestFileFinder.java

```diff
@@ -46,9 +45,11 @@ public InputFile getUnitTestResource(FileSystem fileSystem, String classname) {
          * Most xcodebuild JUnit parsers don't include the path to the class in the class field, so search for it if it
          * wasn't found in the root.
          */
+        String[] fileNameComponents = fileName.split("/");
+        String lastFileNameComponents = fileNameComponents[fileNameComponents.length - 1];
```

Comment:
Should we prefer to use a safer option like [StringUtils.substringAfterLast](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#substringAfterLast-java.lang.String-java.lang.String-)?
An `IndexOutOfBoundsException` could be raised as no check is performed.
The variable should also be checked before used in the `matchesPathPattern` method.

Reply:
Updated with your suggestion.